src/smartdc/bin/sdc-factoryreset.sh

```diff
@@ -27,28 +28,41 @@ function abort()
 
 function usage()
 {
-	printf "\nUsage: $myname [-h | --help]\n\n"
+	printf "\nUsage: $myname [-h | --help] {-s}\n\n"
 	printf "Resets a machine to its originally installed state.  See "
 	printf "sdc-factoryreset(1)\nfor more information.\n"
 	exit 1
 }
 
+function wipe_bootpools()
+{
+	# disable booting on all pools -- XXX KEBE ASKS, make this optional?
+	for pool in $(piadm bootable | grep -v non-bootable | awk '{print $1}')
+	do
+		piadm bootable -d $pool
+	done
+}
+
```

Comment:
I'm very undecided as to whether I think this is ok here or if it should really be done in fs-joyent along with the rest of the pool resets.

Reply:
Me too, given work-Slack conversations (sorry folks in the open-source audience).

This is why 8264 was not examined a lot since its filing...  we may want to split out 8578, and maybe have "wipe bootable now" in `sdc-factoryreset` as its own non-default option as part of 8578, and fix 8264 in `fs-joyent`.